ResidentsSocialCarePlatformApi/V1/Gateways/MosaicGateway.cs

```diff
@@ -21,7 +21,7 @@ public MosaicGateway(MosaicContext mosaicContext)
             _mosaicContext = mosaicContext;
         }
 
-        public List<ResidentInformation> GetAllResidents(int cursor, int limit, long? id = null, string firstname = null,
+        public List<Domain.ResidentInformation> GetAllResidents(int cursor, int limit, long? id = null, string firstname = null,
```

Comment:
Did `Domain` come from the auto refactoring? Do we need it?

Reply:
Not sure why this changed 🤔
We might need the `Domain` prefix but will have a look.